jkube-kit/resource/helm/src/test/java/org/eclipse/jkube/kit/resource/helm/HelmConfigTest.java

```diff
@@ -140,6 +140,36 @@ void deserialize() throws Exception {
         .toBuilder().name("ngnix").version("1.2.3").repository("https://example.com/charts").build()));
   }
 
+  @Test
+  void canEnableCasePresevation() throws IOException {
+    // Given
+    HelmConfig helmConfig = new HelmConfig();
+    helmConfig.setPreserveParameterCase(true);
+
+    File file = File.createTempFile("case-preservation-test", ".tmp");
+    file.deleteOnExit();
+
+    // Then
+    assertThat(helmConfig.isPreserveParameterCase()).isTrue();
+  }
+
+  @Test
+  void canSetApiVersion() throws IOException {
+    // Given
+    HelmConfig helmConfig = new HelmConfig();
+    helmConfig.setApiVersion("v2");
+
+    File file = File.createTempFile("api-version-test", ".tmp");
+    file.deleteOnExit();
```

Comment:
What's the significance of creating this file in scope of this test?

Reply:
Leftover that I missed, removed it.